src/test/DeltaNeutralLp.t.sol

```diff
@@ -103,12 +105,53 @@ contract L1DeltaNeutralTest is TestPlus {
         assertApproxEqRel(assetsLP, assetsInAAve * 2, 0.01e18);
     }
 
+    /**
+     * @notice test start position with less assets than balance
+     */
+    function testStartPositionWithLessAmount() public {
+        uint256 reducedAmount = 10_000;
```

Comment:
Maybe a more realistic number like 10e6 would be better? The strategy has `1000e6` of `asset`